src/docs/maturity/compliance.md

```diff
@@ -24,7 +24,7 @@ Railway follows a shared responsibility model for HIPAA compliance. Railway will
 
 If your company needs a BAA, you can contact our solutions team at [team@railway.app](mailto:team@railway.app).
 
-We are working on operationalized BAAs and continually gathering requirements for health focused workloads for Enterprises. You can subscribe to the road-map item and share your feedback on [our road-map board](https://railway.canny.io/feature-requests/p/compliance).
+We are working on operationalized BAAs and continually gather requirements for health focused workloads on Enterprises. You can subscribe to the road-map item and share your feedback [here](https://railway.canny.io/feature-requests/p/compliance).
```

Comment:
Similar here.  I think it's a good flag, but not sure about the suggested fix.  Maybe these are ones we override with our own fix and see if it passes on the next run?